javascripts/tabs.js

```diff
@@ -1,6 +1,6 @@
 $(function () {
   function selectTab(tab, $parent) {
-    $parent.find('[data-tab="' + tab + '"]')
+    $("li." + tab)
```

Comment:
@tonsV2 Ok, this is a good start, but there is a small problem with this approach. The tabs component consists of 2 things:

- The `ul` element, which contains the tab titles.
- A number of `div`s which contain the content of each tab.

The `data-tab` attribute links the appropriate title with the appropriate div containing the content of this tab. In order to sync all the tabs we need to do something like this:

``` javascript
$('[data-tab="' + tab + '"]')
```
Hope this makes sense. Please update the pull request and reach out to me anytime if you would like to discuss this further. Thanks!